pkg/distributor/distributor_test.go

```diff
@@ -2996,28 +3332,25 @@ func TestDistributor_LabelValuesCardinality(t *testing.T) {
 	}
 
 	tests := map[string]struct {
-		labelNames                []model.LabelName
-		matchers                  []*labels.Matcher
-		ingestersSeriesCountTotal uint64
```

Comment:
Note to reviewers: I've modified this test to not mock the result of `mockIngester.UserStats()` anymore, but having it computing the actual number of pushed series.